src/components/Header.jsx

```diff
@@ -51,12 +59,14 @@ Header.propTypes = {
   experimentData: PropTypes.object,
   title: PropTypes.string.isRequired,
   extra: PropTypes.node,
+  loader: PropTypes.bool,
 };
 
 Header.defaultProps = {
   experimentId: null,
   experimentData: null,
   extra: <></>,
+  loader: false,
```

Comment:
the name `loader` doesn't sound like a boolean. I suggest you rename it to `isLoading` or `loading` instead

Reply:
changed